src/opm/input/eclipse/Schedule/Well/WellConnections.cpp

```diff
@@ -492,6 +509,9 @@ The cell ({},{},{}) in well {} is not active and the connection will be ignored)
             // PolymerMW module.
             ctf_props.re = std::sqrt(D[0] * D[1] / angle * 2);
 
+            ctf_props.static_dfac_corr_coeff =
+                staticForchheimerCoeffcient(ctf_props, props->poro, wdfac);
```

Comment:
Coefficient

Reply:
> Coefficient

Good catch.  I've pushed an update to address the misprint.